### ISSUE TITLE
space key / dispatch 'Space'

### ISSUE THREAD

**Issue by notYou263** [metonym/svelte-keydown#9]
The keyboard event for space key is
`KeyboardEvent: key=' ' | code='Space'`

This leads to call of dispatch with key value ' '  for space key.
Mapping it to 'Space' would be great so one can use on:Space on a component as like for Enter and Escape

**Comment by notYou263** [comment]
From Svelte documentation (https://svelte.dev/docs#run-time-svelte-createeventdispatcher)

> Component events created with createEventDispatcher create a CustomEvent. These events do not bubble and are not cancellable with event.preventDefault()

However, if you pass a function which calls the original keyboard events preventDefault , you can use it from
svelte-keydown event handler. I just missed that one to prevent site scrolling on space key

_Keydown.svelte_


```
  on:keydown={(evt) => {
    let { key, target, preventDefault } = evt
...

  dispatch("key",   { key, preventDefault: () => evt.preventDefault() });
}
```

_App.svelte_
```

<Keydown
  pauseOnInput
  on:key={(e) => {
      e.detail.preventDefault()

      console.log("Keydown event, key: ", e.detail.key)
  }}
/>
```

something like that

**Comment by metonym** [comment]
@notYou263 Nice – I think that's a separate feature request though.

**Comment by metonym** [comment]
@notYou263 The `on:Space` is fixed in v0.4.1. I've opened #11 to track the other feature request.
